nixos/modules/services/misc/rmfakecloud.nix

```
@@ -136,8 +143,8 @@ in {
         RestrictRealtime = true;
         RestrictSUIDSGID = true;
         SystemCallArchitectures = "native";
-        WorkingDirectory = serviceDataDir;
-        StateDirectory = baseNameOf serviceDataDir;
+        WorkingDirectory = cfg.serviceDataDir;
+        StateDirectory = baseNameOf cfg.serviceDataDir;
```

This doesn't work when a path outside of `/var/lib/` is specified.
```suggestion
        StateDirectory = mkIf (cfg.serviceDataDir == "/var/lib/rmfakecloud) "rmfakecloud";`
```

---

@aanderse Right, thanks for the review and post! I had originally tested this with another subdirectory in /var/lib, but ran into this when i tried to use another path. Marked the PR as a draft, but then thought that it's better than the status quo. But we should probably find a better solution before merge.